terraform/modules/stack/messaging.tf

```diff
@@ -315,6 +315,12 @@ module "bag_register_output_topic" {
   ]
 }
 
+resource "aws_sns_topic_policy" "register_topic_subscribe_policy" {
```

Comment:
Worth calling it `bag_register_...` as I read it below and was trying to figure out what it was.